src/render/tutorials/caching-http-requests/2_0.html.md

```diff
@@ -0,0 +1,335 @@
+
+Project has the following structure:
+```
+├── docker
+|   ├── Dockerfile                // Docker file with image definition.
+├── functional                    // Keep here your functional tests. Example implementation included
+├── gradle                        // Gradle wrapper and common gradle scripts
+├── knotx/conf                    // Knotx configuration which will be copied to docker image
+├── modules                       // Sub-modules of your project
+│   ├── ...                       // example modules implementation
+```
+
+## Project configuration
+The first thing we need to do is to specify configuration files. Let's start in the project level directory and simply create `api-cache.yml`
+The file should contains
+``` yml
+version: '3.7'
+
+networks:
+  knotnet:
+
+services:
+
+  webapi:
+    image: rodolpheche/wiremock
+    volumes:
+      - "./common-services/webapi:/home/wiremock"
+    ports:
+      - "3000:8080"
+    networks:
+      - knotnet
+
+  knotx:
+    image: knotx-example/api-cache:latest
+    command: ["knotx", "run-knotx"]
+    ports:
+      - "8092:8092"
+      - "18092:18092"
+    networks:
+      - knotnet
+```
+
+First and foremost it specifies docker virtual network in which our services will be deployed. Then we have some service configs.
+The first one is `webapi`, `image` - docker image is provided here, `volumes` - directories to which Docker will have access, 
+in our case it's directory where all Wiremock configs are stored,
+`ports` - on the left side is provided port accessible from `localhost`, and on the right side we have port number available in 
+virtual environment. The last param `networks` specifies in which docker network service will be deployed.
+Then we have `knotx` service specification, it's very similar to previous one. `command` property is used to point Knot.x starting commands.
+
+Next step in our configuration is to properly set up `settings.gradle.kts` (also at the project level directory) 
+```gradle
+rootProject.name = "api-cache"
+
+include("health-check")
+
+project(":health-check").projectDir = file("modules/health-check")
+``` 
+
+Here we have to specify project name, and packages to be included in this project. 
+
+Next thing to do is to configure `gradle.properties` at project level directory. Let's fill it with
+```gradle
+version=2.0.0-SNAPSHOT
+knotx.version=2.0.0
+knotx.conf=knotx
+docker.image.name=knotx-example/api-cache
+```
+Very important step here is to set `docker.image.name`, which is necessary to run proper docker image.
+
+`build.gradle.kts` file could stay as it was provided with starterkit, we don't have to apply any modifications here.
+The same if it comes to `./docker/Dockerfile`, version provided by default is absolutely enough for our purposes.
+
+## Modules
+Now we can focus on modules - with starterkit some are provided. Very important is `health-check`. Role of this module is quite 
```

Comment:
`healthcheck` is already described in _Getting started with Docker_ tutorial, you can simply provide a link